modules/admin_manual/pages/installation/deployment_recommendations.adoc

```diff
@@ -150,7 +152,7 @@ for information on configuring ownCloud to use LDAP and AD.
 Local session management on the application server. PHP sessions are
 stored in a temporary filesystem, mounted at the operating
 system-specific session storage location. You can find out where that is
-by running `grep -R 'session.save_path' /etc/php5` and then add it to
+by running `grep -R 'session.save_path' /etc/php*` and then add it to
 the `/etc/fstab` file, for example:
 
 [source,console]
```

Comment:
pls check the command below, it is referencing to php5
Same in line about 264 and 274 (maybe shifted a bit but you will find it)

Reply:
@JKawohl pls check the command
`echo "tmpfs /var/lib/php5/pool-www tmpfs defaults,noatime,mode=1777 0 0" >> /etc/fstab`
It is on two locations and contains a reference to php5 which should be at least php7.2 or php7.3

BTW, the path `/var/lib/php5/pool-www` is not valid for Ubuntu, this should be `/etc/php7.2/` but I do not know the subdirectory where `/pool-www` is located because I am running nginx with fpm and not apache.

As a solution, maybe to write it more neutral, do a grep in `/etc/php7.` to find where `pool-www` is located and suggest to use that found path in the command above.

Reply:
> @JKawohl pls check the commands
> `echo "tmpfs /var/lib/php5/pool-www tmpfs defaults,noatime,mode=1777 0 0" >> /etc/fstab`
> It is on two locations and contains a reference to php5 which should be at least php7.2 or php7.3

PHP-FPM should not be mentioned in the docs at all, as it is unsupported.
 
> BTW, the path `/var/lib/php5/pool-www` is not valid for Ubuntu (which we usually use for examples), this should be `/etc/php7.3/` but I do not know the subdirectory where `/pool-www` is located because I am running nginx with fpm which has a different sub location and not apache.
> 
> As a solution, maybe to write it more neutral, do a search in `/etc/php7.*` to find where `pool-www` is located like `find /etc/php7.* -type d | grep "pool-www"` and suggest to use the found path in the command above.

As far as I know the correct path for PHP configuration in Ubuntu version 18.04 and higher is `/etc/php/7.x/`

This whole document needs to be reworked, apparently. Perhaps we quickly apply the changes @JKawohl  suggested, so the really big mistakes are gone, and then we ask the doc-team to go over this document.

Reply:
> PHP-FPM should not be mentioned in the docs at all, as it is unsupported.

Not inteded to be mentioned. I just wrote about my environment and because I do not have Apache, I can't prove the location of `pool-www`

Apparrently the command paths are wrong. Ok to fix this in a separate PR.
Can you pls open a issue and add a link to this comment block, so it is exactly known what it is about.

Reply:
Thanks for the discussion here, people. I'll get the relevant changes implemented.